IPython/html/widgets/tests/test_interaction.py

```diff
@@ -80,7 +80,8 @@ def check_widgets(container, **to_check):
     # build a widget dictionary, so it matches
     widgets = {}
     for w in container.children:
-        widgets[w.description] = w
+        if hasattr(w, 'description'):
+            widgets[w.description] = w
```

Comment:
It shouldn't be a problem, widgets without descriptions cause this code (without the check) to barf, so they can't be included as-is.